examples/de-emphasize.rs

```diff
@@ -0,0 +1,88 @@
+extern crate mdbook;
+extern crate pulldown_cmark;
+extern crate pulldown_cmark_to_cmark;
+
+// This program removes all forms of emphasis from the markdown of the book.
+
+use mdbook::errors::Error;
+use mdbook::MDBook;
+use mdbook::book::{Book, BookItem};
+use mdbook::preprocess::{Preprocessor, PreprocessorContext};
+use pulldown_cmark::{Event, Parser, Tag};
+use pulldown_cmark_to_cmark::fmt::cmark;
+
+use std::ffi::OsString;
+use std::env::{args, args_os};
+use std::process;
+
+struct Deemphasize;
+
+impl Preprocessor for Deemphasize {
+    fn name(&self) -> &str {
+        "md-links-to-html-links"
+    }
+
+    fn run(&self, _ctx: &PreprocessorContext, book: &mut Book) -> ::std::result::Result<(), Error> {
```

Comment:
If you glob import mdbook's errors(`use mdbook::errors::*`) you won't need to fully qualify the `::std::result::Result<Error>` here. It's a personal style thing, but I tend to find fully qualified names less readable.

Reply:
Done! Except that I just import `Result` as I think spelling it out is preferable if the item count is low.